GitDepend.IntegrationTests/Scenarios/HelpPageScenario.cs

```diff
@@ -30,21 +57,47 @@ public void InvalidVerb_Returns_InvalidCommand()
         [Test]
         public void CloneVerb_ShouldCloneAllRepos()
         {
-            var path = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
-            var lib2Dir = Path.Combine(path, Lib2Name);
-            var lib1Dir = Path.Combine(path, Lib2Name);
-            Clone(Lib2Url, lib2Dir);
+            var info = GitDepend("clone", _lib2Dir);
 
-            var info = GitDepend("clone", lib2Dir);
+            var lib1Exists = Directory.Exists(_lib1Dir);
+            var lib2Exists = Directory.Exists(_lib2Dir);
 
-            var lib1Exists = Directory.Exists(lib1Dir);
-            var lib2Exists = Directory.Exists(lib2Dir);
 
-            SafeDeleteDirectory(path);
+            Assert.AreEqual(ReturnCode.Success, info.ReturnCode, "Return Code was Wrong");
+            Assert.IsTrue(lib1Exists, "Lib1 Doesn't Exists");
+            Assert.IsTrue(lib2Exists, "Lib2 Doesn't Exists");
+        }
 
-            Assert.AreEqual(ReturnCode.Success, info.ReturnCode);
-            Assert.IsTrue(lib1Exists);
-            Assert.IsTrue(lib2Exists);
+
+        [Test]
+        public void UpdateVerb_CloneReposChangeLib1_ShouldUpdateArtifacts()
```

Comment:
This is all in the HelpPageScenario, which should focus only on Help page work flows. You should create a new Scenario file for Updating.